frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -23,11 +23,14 @@ function GameStatistics({ finalScores, setIsStatisticsDisplayed, handleResetGame
         <div className="table-item">score</div>
         <div className="table-item">{finalScores.white}</div>
       </div>
+
       <Button
```

Comment:
It would be nice to disable the buttons based on the current application state - '+', '-' or 'Reset' when the game is not running, 'New game' when the game is on already etc.

Reply:
We will make it in another PR to avoid adding too much content in one PR.